tests/test_hull.py

```diff
@@ -18,7 +18,7 @@ def test_hull():
 
     h = hull.find_hull(edges)
 
-    assert len(h.Vertices()) == 11
+    assert len(h.Vertices()) == 10
```

Comment:
Somehow my changes made this hull have one fewer vertices. Based on how the points and arcs are laid out, I'm guessing there were multiple possible next entities with the same angle and my hull changes made it skip one.

There should probably be a 'tie-breaker' for choosing the smallest angle so that the system is more deterministic. (either always choosing the closer to ensure the loop always ends up closed, or choosing the further, but we'd have to be more careful about choosing the starting entity, lest we start on an entity we later skip and fail to close the loop)